common/autoware_auto_perception_rviz_plugin/src/object_detection/predicted_objects_display.cpp

```diff
@@ -23,163 +26,235 @@ namespace rviz_plugins
 {
 namespace object_detection
 {
+void PredictedObjectsDisplay::parallelizedCreateMarkerWorkerThread(int rank)
+{
+  while (true) {                           // keep the thread alive
+    sem_wait(&starting_semaphores[rank]);  // wait for the signal to start
+
+    // Determine the number of objects to be processed by this thread
+    int length = tmp_msg->objects.size();
+    int num_object_per_thread = static_cast<int>(ceil(length / static_cast<float>(num_threads)));
+
+    int first_object_index = rank * num_object_per_thread;
+    int last_object_index = first_object_index + num_object_per_thread;
+    if (last_object_index > length) {
+      last_object_index = length;
+    }
+    std::vector<visualization_msgs::msg::Marker::SharedPtr> Thread_Markers;
```

Comment:
Do you have any reason to use uppercase letter?

Reply:
It was a copy from other sources, I should fix it here.